Dockerfile

```diff
@@ -2,39 +2,26 @@
 FROM python:3.9-slim
 RUN apt-get update \
     && dpkg --add-architecture arm64 \
-    && apt-get install -y --no-install-recommends git ssh socat wget curl libcairo2 python3-dev libffi-dev socat \
-    && apt-get purge -y --auto-remove \
+    && apt-get install -y --no-install-recommends git ssh socat wget curl libcairo2 python3-dev libffi-dev socat gcc \
+    && pip3 install --no-cache-dir --upgrade pip \
+    && pip3 install --no-cache-dir ruamel.yaml.clib \
```

Comment:
this is can be deleted as well. and so is the GCC in install and purge line

Reply:
Sure